contents/docs/user-guides/sessions.md

```diff
@@ -4,4 +4,75 @@ sidebar: Docs
 showTitle: true
 ---
 
-We have removed the Sessions page, as it was confusing users. You can read more on [our blog](../../blog/sessions-removal) about why we did this. 
```

Comment:
Probs worth deleting this blog as well?: https://posthog.com/blog/sessions-removal - or updating the top to say we've brought sessions back!

Reply:
Good point. I've added a comment to that article, via this PR.